ViewControllers/StockFeedViewController.m

```diff
@@ -18,15 +18,15 @@
 
 
 @interface StockFeedViewController () <UISearchBarDelegate, StockDetailsViewDelegate, UITableViewDataSource, UITableViewDelegate>
-@property (weak, nonatomic) IBOutlet UITableView *stockTableView;
-@property (nonatomic, strong)NSMutableArray *stocksArray;
-@property (nonatomic, strong)NSMutableArray *cacheOfInterestedStocks;
-@property (weak, nonatomic) IBOutlet UILabel *todaysDate;
-@property (nonatomic, strong) IBOutlet UIRefreshControl *refresh;
-@property (weak, nonatomic) IBOutlet UISearchBar *searchBar;
-@property (nonatomic, strong)NSMutableArray *filteredStocksArray;
-@property (nonatomic, assign) BOOL isFiltered;
 
+    @property (weak, nonatomic) IBOutlet UITableView *stockTableView;
```

Comment:
For all these `@property` uses you can remove the left space and there should be a space between the different parts i.e.
```
@property (nonatomic, strong) NSMutableArray *stocksArray;
```
instead of
```
    @property (nonatomic, strong)NSMutableArray *stocksArray;
```